### ISSUE TITLE
[Feature Request] Play audio file inside chat (just like voice messages)

### ISSUE THREAD

**Issue by Thijsku** [DrKLO/Telegram#1116]
I think the way audio files are being shared could be a bit more convenient. When an user receives an audio file, he has to open another media player application to be able to hear the file. Wouldn't it be nicer to be able to play an audio file in the chat instead, just the way voice messages are handled?

This approach makes sharing audio files easier, as well as more fun. For example, you can fool around with audio files just like sending gifs. (applause tracks, awkward cricket sound effects, movie quotes.. you name it)

I'd love your opinions on this matter!

![image](https://cloud.githubusercontent.com/assets/3017676/7790182/3b08db9c-027e-11e5-9c22-1d3295fd681c.png)

**Comment by rickkuilman** [comment]
I'd also like to see this, it's a handy addition :+1:

**Comment by brunobsouzac** [comment]
I agree, it would be nice, but it has to have a state that is a different audio of the common voice. Many people may not like.

But I would very much have the option of voice messages with auto play, which only has the iOS. And that the chat had disabling.

Something like this in the Inbox

![google-inbox](https://cloud.githubusercontent.com/assets/12419098/7949621/10e0a8f6-0966-11e5-87a0-932fdb2a77b3.jpg)

**Comment by schlypel** [comment]
I am not quite sure if i did actually understand the feature requested here right, are you proposing automatic playback of received audio messages in a chat?
If so, i definitely give a   :+1: 
If not, i will add a feature request for that one.
I use the "send audio" feature a lot lately, its my PTT replacement :D 
but would be so much easier to use if received audios would just autoplay if chat is currently active.

**Comment by brunobsouzac** [comment]
@schlypel  Hello, just that!

Automatic audio feature is the same that exists for iOS.

But this feature is hidden. Android still there is no appeal.

Option to enable / disable the chat screen would be great.

Enable the volume button - to hold and sending voice message would be perfect.

it would replace the PTT Voxer / Zello I use, it is better than Whatsapp connection feature.

**Comment by brunobsouzac** [comment]
@DrKLO
